verification/curator-service/ui/src/components/ViewCase.tsx

```diff
@@ -133,345 +144,485 @@ function dateRange(range?: { start?: string; end?: string }): string {
 
 function CaseDetails(props: CaseDetailsProps): JSX.Element {
     const classes = useStyles();
+    const scrollTo = (name: string): void => {
+        Scroll.scroller.scrollTo(name, {
+            duration: 100,
+            smooth: true,
+            offset: -64, // Account for header height
+            containerId: 'scroll-container',
+        });
+    };
     return (
-        <Container maxWidth="md" className={classes.container}>
-            <Typography className={classes.caseTitle} variant="h5">
-                Case {props.c._id}{' '}
-                {props.enableEdit && (
-                    <Link
-                        to={`/cases/edit/${props.c._id}`}
-                        style={{ textDecoration: 'none' }}
-                    >
-                        <Button
-                            variant="outlined"
-                            color="primary"
-                            className={classes.editBtn}
-                            endIcon={<EditIcon />}
+        <>
+            <nav className={classes.navMenu}>
+                <Button
+                    variant="text"
+                    onClick={(): void => scrollTo('case-data')}
+                >
+                    case data
+                </Button>
+                <br />
+                <Button
+                    variant="text"
+                    onClick={(): void => scrollTo('demographics')}
+                >
+                    demographics
+                </Button>
+                <br />
+                <Button
+                    variant="text"
+                    onClick={(): void => scrollTo('location')}
+                >
+                    location
+                </Button>
+                <br />
+                <Button
+                    variant="text"
+                    onClick={(): void => scrollTo('event-history')}
+                >
+                    event history
+                </Button>
+                <br />
+                <Button
+                    variant="text"
+                    onClick={(): void => scrollTo('symptoms')}
+                >
+                    symptoms
+                </Button>
+                <br />
+                <Button
+                    variant="text"
+                    onClick={(): void => scrollTo('travel-history')}
+                >
+                    travel history
+                </Button>
+                <br />
+                <Button
+                    variant="text"
+                    onClick={(): void => scrollTo('pathogens')}
+                >
+                    pathogens
+                </Button>
+                <br />
+                <Button variant="text" onClick={(): void => scrollTo('notes')}>
+                    notes
```

Comment:
To match the sections that currently exist in the details page I think these titles should be (missing/differing titles bolded):
case data
demographics
location
event history,
symptoms
**preexisting conditions**
**transmission**
travel history
**pathogens & genome sequencing**
notes

I would also split up pathogens and genome sequencing since they aren't related. Pathogens is other pathogens the case has and genome sequencing is pertaining to the disease in question (covid). But happy for that to be in a different PR since this is just about adding the section titles.